modules/luxon.js

```diff
@@ -0,0 +1,5 @@
+const DateNowString = () => {
+  const date = new Date(Date.now());
+  return `${date.toUTCString().slice(0, 16)}`;
+};
+export default DateNowString;
```

Comment:
I can't see any usage of `luxon` package here! :thinking: 
Also I can't see any existence of `luxon` node installed in the packages manager `npm`

Go ahead and try to follow the instruction of `luxon` documentation to satisfy this requirement [LINK](https://moment.github.io/luxon/#/install?id=nodejs)